Website/DesktopModules/Hotcakes/Hotcakes.Modules.csproj

```diff
@@ -135,14 +138,24 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\References\Misc\paypal_base.dll</HintPath>
     </Reference>
+    <Reference Include="Stripe.net, Version=40.9.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\References\Misc\Stripe.net.dll</HintPath>
```

Comment:
In other instances, this is coming from Nuget.  Why is this one coming from the `References` path?  Is there something that copies the Nuget DLLs to that folder or something?

Reply:
In the current version, this dll comes from the References folder. I assume they compiled Stripe's solution to generate this solution with all its dependencies in a single dll file. Using the approach via nuget, all dependencies must be added.